prtpy/objectives.py

```diff
@@ -156,6 +155,44 @@ def lower_bound(self, sums:list, sum_of_remaining_items:float, are_sums_in_ascen
 
 
 
+class MinimizeTheDistanceFromAvg(Objective):
+    def value_to_minimize(self, sums:list, are_sums_in_ascending_order:bool=False)->float:
+        avg = sum(sums) / len(sums)
+        diff_from_avg = 0
+        for s in sums:
+            if (s > avg):
+                diff_from_avg = diff_from_avg + (s - avg)
+        return diff_from_avg
+    def __str__(self) -> str:
+        return "minimize-the-distance-from-avg"
+    def lower_bound(self, sums:list, sum_of_remaining_items:float, are_sums_in_ascending_order:bool=False)->float:
+        """
```

Comment:
There should be an explanation to the algorithm

Reply:
I added an explanation for the algorithm.